xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -668,6 +668,14 @@ static void DisplayReconfigured(CGDirectDisplayID display,
       NSString* title = [NSString stringWithUTF8String:m_name.c_str()];
       appWindow.backgroundColor = NSColor.blackColor;
       appWindow.title = title;
+      //! TODO: Remove me if minimum version is bumped to 10_14
+      if (floor(NSAppKitVersionNumber) < NSAppKitVersionNumber10_14)
```

Comment:
Im not sure this will work as intended.

according to https://developer.apple.com/documentation/appkit/nsappkitversionnumber10_14?language=objc this is available in 10.15+

Looking at the headers of SDK's seems to back that up

10.15 - https://github.com/phracker/MacOSX-SDKs/blob/041600eda65c6a668f66cb7d56b7d1da3e8bcc93/MacOSX10.15.sdk/System/Library/Frameworks/AppKit.framework/Versions/C/Headers/NSApplication.h#L79-L83
10.14 - https://github.com/phracker/MacOSX-SDKs/blob/041600eda65c6a668f66cb7d56b7d1da3e8bcc93/MacOSX10.14.sdk/System/Library/Frameworks/AppKit.framework/Versions/C/Headers/NSApplication.h#L72-L76

@kambala-decapitator any idea?

Reply:
not quite sure why you want this condition at all.

> This property does not do anything and should not be used

since it does nothing, it doesn't hurt to have it. Pragmas already silence the warning.

If you really want to ignore it on OS >= 10.15, then it's better to use

```objc
if (![NSProcessInfo.processInfo isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){.majorVersion = 10, .minorVersion = 15, .patchVersion = 0}])
```

You can also wrap the condition with body in

```objc
#if __MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_15
...
#endif
```

to not compile it at all once we raise deployment target to 10.15. Just make sure that `__MAC_10_15` is defined - it's in `AvailabilityVersions.h` but can be missing if SDK used is < 10.15.

Reply:
thanks, I've adjusted accordingly. According to the deprecation notice it states this was deprecated in macOS 10.14 (available but does nothing). I am not sure what the behaviour is before that but I assume it's here for a reason.
Hence, my goal here is to avoid using it (and silence the warning) if the version is higher or equal to 10.14

Reply:
Honestly I don't think it's worth merging unless someone running 10.13 raises an issue that this magically solves.

Reply:
sounds fine to me, I'm only trying to be safe here. 
I doubt anyone running 10.13 will ever run native windowing anyway.